macros/collect_qe_plots.C

```diff
@@ -150,11 +149,11 @@ TStyle* SetPTFStyle(Int_t WhichStyle = 1, TString styleName = "PTF") {
   // - inbuilt color schemes -
   // ptfStyle->SetPalette(1); // use the rainbow color set
   //ptfStyle->SetPalette(kViridis); // use the viridis color set
-  ptfStyle->SetPalette(kSunset); // use the sunset color set
+  //ptfStyle->SetPalette(kSunset); // use the sunset color set
```

Comment:
Just changes in plotting script? Should include such changes in a different PR, or include in the title/description of this one.

Reply:
Probably a mistake, doesn't change anything in the accelerometer, no related.